tests/rustdoc/notable-trait/notable-trait-generics.rs

```diff
@@ -0,0 +1,35 @@
+#![feature(doc_notable_trait)]
+
+// Notable traits SHOULD be shown when the `impl` has a generic type and the
+// return type has a concrete type.
+pub mod generic_return {
+    pub struct Wrapper<T>(T);
+
+    #[doc(notable_trait)]
+    pub trait NotableTrait {}
+
+    impl NotableTrait for Wrapper<u8> {}
+
+    // @has notable_trait_generics/generic_return/fn.returning.html
+    // @!has - '//a[@class="tooltip"]/@data-notable-ty' 'Wrapper<T>'
```

Comment:
Wait, shouldn't this be `@has` (and the one below be `@!has`)?

Reply:
Ah, that makes sense. However, I think your comments above the `pub mod` are the inverse of what they should be? For this one, it says "SHOULD be shown", when in fact it shouldn't. And the opposite scenario for the other module. (Correct me if I misunderstood!)